### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wyatt-webb**
Add instructions for installing and using byobu sessions to run flask web server within.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,7 +194,7 @@ To run the web server in development mode use the following command. This must b
 flask --app sandman_web run --debug --host 0.0.0.0
 ```
 
-Then in your web browser enter the following URL: YOUR_SANDMAN_IP_ADDRESS:5000/reports. Currently you must provide the report suffix, because index redirection is not working at the moment.
+Then in your web browser enter the following URL: YOUR_SANDMAN_IP_ADDRESS:5000/reports. Currently you must provide the report suffix, because index redirection is not working at the moment. You can stop the web server with CTRL + C.
```
Comment (shawn-lindberg) [inline README.md:197]: Now that you have fixed the index redirection, let's change this to just point at the index without the caveat. I would also specify that you are pressing control C in the TERMINAL.

File: README.md [hunk 2]
```diff
@@ -158,8 +158,17 @@ sudo make install
 
 Sandman has a web interface implemented with Flask. These instructions cover what you need to do in order to run this web interface in development mode. In the future there will be a way to run this web interface in a more production friendly environment.
 
-Installation instructions can be found here: [Flask Installation](https://flask.palletsprojects.com/en/3.0.x/installation/). However, they are not complicated and we can skip a step, so they will be provided here. Assuming that your Sandman repository is stored in the home directory, it is recommended to switch to the Sandman web directory in order to set up the virtual Python environment. This is not strictly required, but is recommended by Flask.
+Installation instructions can be found here: [Flask Installation](https://flask.palletsprojects.com/en/3.0.x/installation/). However, they are not complicated and we can skip a step, so they will be provided in the following instructions.
 
+Install byobu
```
Comment (shawn-lindberg) [inline README.md:163]: I didn't know about byobu before, but I took a look to understand what this was. It seems pretty interesting, but definitely not required. I don't think these instructions really make it clear at all that this is completely optional and as such, I don't think we should be including instructions on it. If we want to plug it, that's fine, but I would just say that you might want to consider using it and provide a link to it for people that are curious. In terms of instructions I think we should focus on the minimum number of required steps except in cases where there is some recommendation by the software we are using.

**Review by shawn-lindberg — CHANGES_REQUESTED** [review]
Please see my comments.

File: README.md (hunk 1 above)
Comment (wyatt-webb) [reply README.md]: I agree with these two suggestions. The changes have been committed.

File: README.md (hunk 2 above)
Comment (wyatt-webb) [reply README.md]: I understand and I'm willing to make it clear that using byobu is optional and remove the instructions related to it.

**Pull request merged**